.pre-commit-config.yaml

```diff
@@ -21,6 +21,17 @@ repos:
       - id: flake8
         args:
           - "--config=./tools/flake8.cfg"
+  # Python typing
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.6.0
+    hooks:
+      - id: mypy
+        # When charmory code passes mypy, add this pattern to the list below
```

Comment:
dead comments?

Reply:
Still valid as of this PR. Next one cleans them up and removes this comment.